src/components/proposal/ProposalTableInstrumentScientist.tsx

```diff
@@ -121,14 +119,24 @@ const ProposalTableInstrumentScientist: React.FC<{
     ...DefaultQueryParams,
     call: NumberParam,
     instrument: NumberParam,
-    proposalStatus: withDefault(NumberParam, 2),
+    proposalStatus: withDefault(
+      NumberParam,
+      featureContext.featuresMap.get(FeatureId.PROPOSAL_FILTER)?.isEnabled
+        ? 0
+        : 2
+    ),
     questionId: StringParam,
     compareOperator: StringParam,
     value: StringParam,
     dataType: StringParam,
     reviewModal: NumberParam,
     modalTab: NumberParam,
-    reviewer: defaultReviewerQueryFilter,
+    reviewer: withDefault(
+      StringParam,
+      featureContext.featuresMap.get(FeatureId.PROPOSAL_FILTER)?.isEnabled
```

Comment:
Could a separate setting be better than a feature here, too? The setting could define `DEFAULT_INST_SCI_REVIEWER_FILTER` and map to the names?